py/selenium/webdriver/firefox/firefox_profile.py

```diff
@@ -93,6 +94,13 @@ def update_preferences(self):
                 f.write(f'user_pref("{key}", {json.dumps(value)});\n')
 
     # Properties
+    @property
```

Comment:
Let's just remove all of this, don't need to maintain an extra property for something that is backwards compatible.

Reply:
Okay, now the parameter `strict_timestamps = False` is simply passed in the `encode` property